core-bundle/src/Resources/contao/classes/Versions.php

```diff
@@ -463,8 +463,11 @@ public function compare($blnReturnBuffer=false)
 				$arrOrderFields = $objDcaExtractor->getOrderFields();
 
 				// Find the changed fields and highlight the changes
-				foreach ($to as $k=>$v)
+				foreach (array_keys(array_merge($to, $from)) as $k)
```

Comment:
This is a behavior change, because now the loop also handles fields that are in `$from` but no longer in `$to`. Is this what we want?

Reply:
> Is this what we want?

IMO yes.

Reply:
But that data cannot be restored anyway, so may be it should not be displayed after all?

Reply:
That‘s what I thought.

Reply:
I think this information could be very valuable, especially if I want to debug something and know that an older version has worked previously.

Reply:
Ok, so if `$from['title']` exists and `$to['title']` does not, the field is currently not shown in the diff at all. After the change, it would be shown like this:

<img width="440" alt="" src="https://user-images.githubusercontent.com/1192057/232747069-be0ac573-60ad-464d-ad2c-620f86e86e6e.png">

So I guess @ausi has a good point here.